packages/reactive-dict/reactive-dict.js

```diff
@@ -124,6 +139,47 @@ _.extend(ReactiveDict.prototype, {
     if (_.has(self.keys, key)) oldValue = parse(self.keys[key]);
     return EJSON.equals(oldValue, value);
   },
+  clear: function(key){
+    var self = this;
+    self.set(key, null);
+    return true;
+  },
+  toggle: function(key){
+    var self = this;
+
+    // toggle currently only works on boolean values
```

Comment:
Um...  my experience has been that there are a lot of edge cases around null, undefined, and empty strings.  It's not clear how ``!null`` or ``!undefined`` or ``!""`` should be handled.  What's the negative of a object?  Personally, I never use ``!Session.get("x")`` in my code, precisely because of those edge cases.  So it's not exactly the pattern I was trying to replace when I created this.